docs/what-is-regression-games.mdx

```diff
@@ -2,58 +2,57 @@
 sidebar_label: 'What is Regression Games?'
 ---
 
-# Regression Games for Unity
+# Regression Games
 
-Regression Games is developing a platform where studios can **build and deploy bots for a variety of use cases in minutes.** We are designing for scale (deploy thousands of bots), ease of use (e.g. generative AI to build bots), ease of integration (minimal impact on game development time), and feature richness (e.g. A/B bot test for engagement).
+Regression Games is on a mission to transform the landscape of game development by tackling one of its most persistent challenges: automated testing.
 
-![vision](img/vision.png)
+We are developing the tools and standards that will make effective end-to-end automated testing of games a reality, in a way that actually saves you
+time and pain. We are imagining a future where automated tests can handle the most complex and repetitive scenarios, freeing up valuable time for developers to focus on
+the most challenging and creative aspects of game development. By combining old school algorithms, cutting-edge machine learning techniques, and a focus
+on the developer experience, Regression Games is building tools that make this end-to-end testing possible.
 
-## Current Stage of the Product
+- **Unleash Your Creativity:** By automating complex and repetitive testing scenarios, we free up your time to focus on what truly matters - creating amazing bug-free games.
+- **Cutting-Edge Technology:** We blend traditional algorithms with state-of-the-art machine learning, creating a powerful synergy that tackles the unique challenges of game testing.
+- **Developer-Centric Approach:** Our tools are built by developers, for developers. We understand your pain points and have crafted solutions that seamlessly integrate into your workflow.
 
-This product is currently a **release preview**. We are actively talking with studios and game developers to discover the best features and use cases to tackle, and we're making changes frequently.
+## What is our vision?
```

Comment:
Not sold on this section being titled "vision". It reads more like the previous section is our high-level vision, and this one is more about our core principles for the implementation of that vision. I almost want to combine these sections because I don't see a strong distinction between the two. I'll give this another look once we clarify what these two sections is meant to convey to the reader (or brainstorm some edits with you over slack)

Reply:
Combined